### PR TITLE
layered-config - not just tableread anymore!

### PR DESCRIPTION
**Pull request by dgou**
Tweak some doc cut'n'paste that we missed.
This mostly shows up in the upper left hand corner of:
https://jolly-good-toolbelt.github.io/layered-config/index.html
and built with this PR:
https://dgou.github.io/layered-config/index.html

### CODE REVIEW

**Review by bradsbrown — APPROVED** [review]
Seems appropriate

**Review by brolewis — APPROVED** [review]
👍

File: sphinx_docs/conf.py [hunk 1]
```diff
@@ -31,9 +31,9 @@
 ######################################################
 
 # General information about your project.
-project = "Tableread - turn a rST table into a Python object!"
-copyright = "2018, Brad Brown"  # noqa
-author = "Brad Brown"
+project = "layered-config - library for layering config files"
+copyright = "2018, Doug Philips and Ryan Casperson"  # noqa
```
Comment (rbcasperson) [inline sphinx_docs/conf.py]: No need to add me here. I'd like to be considered a maintainer/contributor, but I say keep it simple and make yourself the main point of contact for this project.

File: sphinx_docs/conf.py [hunk 2]
```diff
@@ -31,9 +31,9 @@
...
+author = "Doug Philips"
```
Comment (rbcasperson) [inline sphinx_docs/conf.py:36]: Go ahead and update the `author` field in `pyproject.toml` to match this related change. Same thing, no need to keep me in that authors list. I'll take the honorary, undocumented title of Editor in Chief 😄.

**Review by rbcasperson — CHANGES_REQUESTED** [review]
Let's just go ahead and keep it simple with you being the single author/point of contact for this project. It is most accurate and is completely good with me.

File: sphinx_docs/conf.py (hunk 2 above)
Comment (dgou) [reply sphinx_docs/conf.py:36]: Ok.

File: CONTRIBUTORS.rst [hunk 3]
```diff
@@ -4,7 +4,7 @@ Contributors
 This package was the result of collaborative dissussion and effort between:
 
 * Brad Brown @bradsbrown
-* Ryan Casperson @rbcasperson
+* Ryan Casperson @rbcasperson (Honorary, "undocumented", Editor-in-Chief)
```
Comment (rbcasperson) [inline CONTRIBUTORS.rst]: 😆

**Review by rbcasperson — APPROVED** [review]

**Pull request merged**